pennylane/measurements/classical_shadow.py

```diff
@@ -493,6 +491,8 @@ class ShadowExpvalMP(MeasurementTransform):
             where the instance has to be identified
     """
 
+    _shortname = ShadowExpval  #! Note: deprecated. Change the value to "shadowexpval" in v0.42
```

Comment:
Where is this `_shortname` used?

Reply:
@astralcai mostly in drawer or circuit graph. Also, MP.__repr__ used `return_type` before, therefore whenever the MP.__repr__ is called it is also used.